sass/custom/lbs.scss

```diff
@@ -0,0 +1,55 @@
+$govuk-font-family: -apple-system,BlinkMacSystemFont,helvetica neue,helvetica,ubuntu,roboto,noto,segoe ui,arial,sans-serif; // Override font as so not to use New Transport
```

Comment:
could you name the file "london-borough-of-sutton.scss" or whatever its called just so it's easier to know who it's for if we need to update or something

Reply:
we already have stuff like `kcc.scss` or whatever so I just copied that